src/test/operators/table_scan_between_test.cpp

```diff
@@ -57,50 +57,91 @@ class TableScanBetweenTest : public TypedOperatorBaseTest {
     _data_table_wrapper = std::make_shared<TableWrapper>(data_table);
     _data_table_wrapper->execute();
   }
+
+  void _test_between_boundaries(std::vector<std::tuple<AllTypeVariant, AllTypeVariant, std::vector<int>>>& tests,
+                                bool left_inclusive = true, bool right_inclusive = true) {
+    const auto& [data_type, encoding, nullable] = GetParam();
+    std::ignore = encoding;
+    resolve_data_type(data_type, [&, nullable = nullable](const auto type) {
+      for (const auto& [left, right, expected_with_null] : tests) {
+        SCOPED_TRACE(std::string("BETWEEN ") + std::to_string(boost::get<double>(left)) +
+                     (left_inclusive ? " (inclusive)" : " (exclusive)") + " AND " +
+                     std::to_string(boost::get<double>(right)) + (right_inclusive ? " (inclusive)" : " (exclusive)"));
```

Comment:
``boost::get<type>`` is used multiple times in this file, why not use ``type_cast_variant<type>``?

Reply:
Because the type is already known, so we can save us some overhead.